CleanerInstaller.sh

```diff
@@ -55,14 +100,25 @@ done
 
 #remove the old Cleaner and Cleaner42 if there are any, then copy the current one to Home dir
 /bin/rm -rf ~/Cleaner_42.sh &>/dev/null
+/bin/rm -rf ~/LinuxCleaner_42.sh &>/dev/null
 /bin/rm -rf ~/Cleaner.sh &>/dev/null
-cp -f ./Cleaner_42.sh "$HOME"
 
-if ! grep "alias cclean='bash ~/Cleaner_42.sh'" <"$shell_f" &>/dev/null; then
-	echo -e "\nalias cclean='bash ~/Cleaner_42.sh'" >>"$shell_f"
+# Copy the appropriate cleaner script to Home dir
+cp -f ./$CLEANER_SCRIPT "$HOME"
+
+# Remove old aliases from shell config
+if [ -f "$shell_f" ]; then
+	# Create a temporary file without the old aliases
+	grep -v "alias cclean=" "$shell_f" > "${shell_f}.tmp" 2>/dev/null
```

Comment:
Better to create the temp file `${shell_f}.tmp` inside `/tmp/` directory to avoid user files.
`sed` command can also be useful here to edit the shell config file, but I believe it works differently on MacOS and Linux, the current way is efficient and simple.